packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -485,4 +485,79 @@ test.describe('PieCookieBanner - Component tests', () => {
             });
         });
     });
+
+    test.describe('`defaultPreferences` : prop', () => {
+        test.describe('when defaultPreferences are set to three values', () => {
+            test('should toggle the position to `on` for properties', async ({ mount, page }) => {
+                await mount(PieCookieBanner, {
+                    props: {
+                        defaultPreferences: {
+                            functional: true,
+                            personalized: true,
+                            analytical: true,
+                        },
+                    } as CookieBannerProps,
+                });
+
+                // Act
+                await page.click(managePreferencesSelector);
+
+                const functional = await page.locator(getPreferenceItemSelector('functional')).isChecked();
+                const personalized = await page.locator(getPreferenceItemSelector('personalized')).isChecked();
+                const analytical = await page.locator(getPreferenceItemSelector('analytical')).isChecked();
```

Comment:
```suggestion
                const functionalEnabled = await page.locator(getPreferenceItemSelector('functional')).isChecked();
                const personalizedEnabled = await page.locator(getPreferenceItemSelector('personalized')).isChecked();
                const analyticalEnabled = await page.locator(getPreferenceItemSelector('analytical')).isChecked();
```

Reply:
(or something similar)

Reply:
You could probably remove the `isChecked` here, and instead, modify your assertion like so:

https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-checked